packages/ripple-nuxt-tide/modules/landing-page/lib/content-collection.js

```diff
@@ -992,13 +992,13 @@ module.exports = class ContentCollection {
     let mappedResult = null
     const _source = item._source
     const link = this.getLocalisedLinkFromSource(_source)
-
+    const newsDate = _source.field_news_date
     switch (this.getDisplayResultComponentType()) {
       case 'search-result':
         mappedResult = {
           title: _source.title?.[0],
           link: link ? { linkText: link.domain + link.path, linkUrl: link.path } : null,
-          date: _source.created?.[0],
+          date: newsDate ? _source.field_news_date?.[0] : _source.created?.[0],
```

Comment:
You're defining `newsDate` above, so you can use it here as well eg.
```
date: newsDate ? newsDate[0] : _source.created?.[0],
```
Otherwise you could drop that const entirely:
```
date: _source.field_news_date ? _source.field_news_date[0] : _source.created?.[0],
```